src/activity_handlers.cpp

```diff
@@ -918,9 +918,12 @@ void activity_handlers::reload_finish( player_activity *act, player *p )
         if(reloadable->is_gun()) {
             islot_gun* gun = reloadable->type->gun.get();
             if( gun->reload_noise_volume > 0 ) {
+
+              sfx::play_variant_sound( "reload", reloadable->typeId(), sfx::get_heard_volume(p->pos()));
               sounds::sound( p->pos(), gun->reload_noise_volume, gun->reload_noise,
-                             true, "reload", reloadable->typeId() );
+                             true, "", "" );
             }
+            sfx::play_variant_sound( "reload", reloadable->typeId(), sfx::get_heard_volume(p->pos()));
```

Comment:
This is called twice, here and inside the `if`, why? Especially since it plays a sound, even if `reload_noise_volume` is 0 (which I assume indicates no reload sound at all).